ElasticBox/reporting.md

```diff
@@ -112,7 +124,15 @@ Cost Centers help you set and enforce a quota on how much users or team workspac
    ![admin-reports14.png](../images/ElasticBox/admin-reports14.png)
 
 ### Contacting ElasticBox Support
+<<<<<<< HEAD
```

Comment:
Clean up this section. remove the `<<<<<<< HEAD`, `=======`, and `>>>>>>> CenturyLinkCloud/master` lines. Also remove the duplicate content.